include/deal.II/distributed/grid_tools.h

```diff
@@ -130,6 +131,17 @@ namespace parallel
     exchange_cell_data_to_ghosts (const MeshType &mesh,
                                   std::function<boost::optional<DataType> (const typename MeshType::active_cell_iterator &)> pack,
                                   std::function<void (const typename MeshType::active_cell_iterator &, const DataType &)> unpack);
+
+    /**
+     * Compute a vector of bounding boxes which encloses the space occupied by
+     * the locally owned cells. The number of bounding boxes (two or more) tries
```

Comment:
So even if the partitioning is such that one bounding box will perfectly describe the domain, I will get two of them?

Reply:
I added a new version where there is a flag to allow or not merging.
If it is not allowed then yes, two bounding boxes shall be returned.
If it is allowed then the two bounding boxes shall be merged if they can be described as one (for instance (0,0) (1,1) and (1,0), (2,1) become (0,0), (2,1)) ).